### PR TITLE
 Presave fill color in uploaded svg files

### LINKED ISSUE #92: Presave fill color in uploaded svg files
Steps:
- using FUT plugin enable uploading SVG files
- using WPForms create a form to upload files, set to store them in WP Media Library
- upload this file
<details>
<summary>MY SVG</summary>

Image

</details>
- check wp media library if the file has preserved colors

### PR DESCRIPTION
**Pull request by kkarpieszuk**
Fixes #92 

Steps:
- using FUT plugin enable uploading SVG files
- using WPForms create a form to upload files, set to store them in WP Media Library
- upload this file
<details>
<summary>MY SVG</summary>

Image

</details>
- check wp media library if the file has preserved colors

### CODE REVIEW

File: src/Sanitizer.php [hunk 1]
```diff
@@ -107,6 +108,52 @@ public function sanitize( string $file, string $file_name ): bool {
...
+	/**
+	 * Add SVG to wp_get_ext_types results when it's enabled.
+	 *
+	 * @since {VERSION}
+	 *
+	 * @param array|mixed $types Array of allowed file types.
+	 *
+	 * @return array
+	 */
+	public function include_svg( $types ): array {
```
Comment (kkarpieszuk) [inline src/Sanitizer.php:120]: it is not new method, I just moved it from the end of file (I like when class has methods grouped by visibility and this public method was just below private ones)

File: src/SanitizerSvg.php [hunk 2]
```diff
@@ -22,41 +22,41 @@ class SanitizerSvg {
 			'width', 'height', 'viewBox', 'xmlns', 'version', 'preserveAspectRatio'
 		],
 		'rect' => [
-			'x', 'y', 'width', 'height', 'rx', 'ry', 'fill', 'stroke', 'stroke-width'
+			'x', 'y', 'width', 'height', 'rx', 'ry', 'fill', 'stroke', 'stroke-width', 'style'
```
Comment (kkarpieszuk) [inline src/SanitizerSvg.php:25]: the SVG from example has color defined like that:

```
<polygon style="fill:#F95428;" points="132.7...
```
so we need to enable `style` attr for kses. and then enable it for filter `safe_style_css` which removes all non-standard CSS properties

**Review by seredniy — APPROVED** [review]
Good job, @kkarpieszuk! LGTM!

**Review by kagg-design — APPROVED** [review]
Thank you, @kkarpieszuk, LGTM!

**Pull request merged**